instrumentation/grpc-1.6/library/src/main/java/io/opentelemetry/instrumentation/grpc/v1_6/GrpcTelemetryBuilder.java

```diff
@@ -115,6 +116,14 @@ public GrpcTelemetryBuilder setCaptureExperimentalSpanAttributes(
     return this;
   }
 
+  /** Sets which metadata request values should be captured as span attributes. */
+  @CanIgnoreReturnValue
+  public GrpcTelemetryBuilder setRequestMetadataValuesToCapture(
```

Comment:
How about naming it
```suggestion
  public GrpcTelemetryBuilder setCapturedRequestMetadata(
```
? It's similar to the already existing HTTP headers feature